src/plugins/discover/public/plugin.tsx

```diff
@@ -79,6 +80,7 @@ import {
   DiscoverContainerInternal,
   type DiscoverContainerProps,
 } from './components/discover_container';
+import { getSearchProvider } from './globar_search/search_provider';
```

Comment:
```suggestion
import { getSearchProvider } from './global_search/search_provider';
```
I think this folder should be `global_search` instead, although `globar` kinda sounds like a cool name for a global search bar tbh 😄

Reply:
Let's add some r .... globarrrrrrsearch 🦁 !

Reply:
Ooops!